test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -15,33 +15,32 @@
  * limitations under the License.
  */
 
-#include <boost/filesystem.hpp>
-#include <boost/filesystem/fstream.hpp>
-
 #include "ametsuchi/impl/flat_file/flat_file.hpp"
-
 #include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include <boost/filesystem/fstream.hpp>
 #include "common/files.hpp"
 #include "common/types.hpp"
 
 #include "logger/logger.hpp"
 
 using namespace iroha::ametsuchi;
+namespace fs = boost::filesystem;
 using Identifier = FlatFile::Identifier;
 
 static logger::Logger log_ = logger::testLog("BlockStore");
 
 class BlStore_Test : public ::testing::Test {
  protected:
   void SetUp() override {
-    mkdir(block_store_path.c_str(), S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
+    fs::create_directory(block_store_path);
     block = std::vector<uint8_t>(100000, 5);
   }
   void TearDown() override {
-    iroha::remove_all(block_store_path);
-    rmdir(block_store_path.c_str());
+    fs::remove_all(block_store_path);
   }
-  std::string block_store_path = "/tmp/dump";
+  std::string block_store_path =
```

Comment:
you can use fs::path instead of string, you do not need to cast `<...>.string()` every time